src/components/Header/Header.scss

```diff
@@ -2,11 +2,42 @@
 header {
     font-family:$font-text;
     background:$color-light;
+    padding:10px 0;
+    border-bottom: 1px solid #ebebeb;
+    width:700px;
+    width:calc(100% - 35px);
+    display: inline-block;
+   
 }
+
 .logo {
-    display: inline-block;
+    height: 500px;
     padding-top:0;
-    max-height:300px;
+    font-size:100%;
     max-width:100%;
+    position:relative;
+    text-align:center;
+    float:left;
+    vertical-align: middle;
+    margin-top:-70px;
+    
+    
     
-    }
\ No newline at end of file
+}
+.icons {
+    float:right;
+    height:30px;
+    width:30px;
+    display:block;
+    margin-right:30px;
+    margin-top:10px;
+    
+
+}
+input{
```

Comment:
tutaj musimy się dogadać, bo gdybym i ja w swoim komponencie stylowała sam input, to któryś by się nadpisał. Czy nie lepiej byłoby nadać klasę .header dla całego komponentu i zawrzeć go w niej, coś takiego
.header {
 &input {
 ...
}
}?